contrib/containerlab/bgpv2/multi-homing/Makefile

```diff
@@ -0,0 +1,20 @@
+deploy:
+	kind create cluster --config cluster.yaml
+	sudo containerlab -t topo.yaml deploy
```

Comment:
could add a check for containerlab to print a more helpful message from where to install it if one doesn't yet have it

Reply:
I wanted to keep these make files simple. I added a readme section under containerlab/bgpv2, which provides prerequisites of installing kind, containterlab etc. Hope that is enough for users to know what kind of binaries are required to run the lab. I think target audience probably already have container lab - if not, readme should help.